guides/advanced_guides/filtering.md

```diff
@@ -0,0 +1,131 @@
+# Filtering
+
+A search filter is a parameter that can be applied to the search query to refine the search results of a particular field (e.g. by size, color, price, or brand).
+Multiple filters may be applied to take a broad range of documents and refine them into a more narrow selection, allowing the end user to retrieve the most relevant search results based on the criteria they've selected.
+**MeiliSearch supports filtering through a simple query language.**
+
+### The Query Language
+
+In itself the query language is very simple, and allows you to filter results on any document field. For now it only allows you to query on fields that are either `number`, `boolean` or `string`
+
+### Conditions
+
+Conditions are the primitives of query filters. They are triplets `field OP value` where:
+
+- `field` refers to the document attribute to filter on. (_e.g_ id, title...). It is either a single alphanumeric word or a quoted string: `"movie title"`, and `movie_title` are both valid `field`.
```

Comment:
Does this means that if I have an attribute composed of multiple separated words they can be represented without quotes by adding a `_` between each word? Just to be sure I understood correctly

Reply:
no, the id must be the same as the one in the document. But if you want to filter on a key that's multiple words you ought to use quotes. Any suggestion on how to make it clearer?

Reply:
By using two different attributes we don't think it's two different way to format the same information. But rather two information formatted differently that works the way you display it.
```suggestion
- `field` refers to the document attribute to filter on. (_e.g_ id, title...). It is either a single alphanumeric word or a quoted string: `"movie title"`, and `release_date` are both valid `field`.
```